charts/prometheus-stackdriver-exporter/templates/servicemonitor.yaml

```
@@ -22,6 +22,14 @@ spec:
     scrapeTimeout: {{ .Values.serviceMonitor.scrapeTimeout }}
     {{- end }}
     honorLabels: {{ .Values.serviceMonitor.honorLabels }}
+    {{- if .Values.serviceMonitor.metricRelabelings }}
+    metricRelabelings:
+{{ toYaml .Values.serviceMonitor.metricRelabelings | indent 4 }}
```

```suggestion
    {{ toYaml .Values.serviceMonitor.metricRelabelings | nindent 4 }}
```
Could you check if that also works?

---

yeap, it works, also trimming whitespaces gives a cleaner template output... But then also there are a dozen of those occurrences in the Chart. WDYT, it worth changing those also?

---

Sure maybe in a separate PR?

---

@torstenwalter changed on this PR. Seems to be just it.